include/itkPhaseCorrelationOptimizer.hxx

```diff
@@ -20,6 +20,36 @@
 
 #include "itkPhaseCorrelationOptimizer.h"
 
+#ifndef NDEBUG
```

Comment:
Debug

Reply:
WriteDebug will have to stay until the whole code is well debugged. We will need it in the future too.